RELEASE_NOTES.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+
+## 2.0.0 Release Notes Summary
+
+A summary of changes in the 2.0.0 version of dxCompiler comparing to the previous dxWDL version 1:
+
+User-facing changes:
+* Changed the name and all references from dxWDL to dxCompiler, dxCompiler-\*.jar is now used to compile an application or a workflow
+* Replaced Cromwell WOM (parser, type checker and evaluator) with [wdlTools](https://github.com/dnanexus-rnd/wdlTools), a library maintained by DNAnexus
+* Added support for per-workflow and per-scatter chunk size settings to extras.json
+* Added `streamFiles` compile option
+* Updated the mechanism of comparing instance types to select the cheapest one for execution
+* Improved file resolving and caching
+* Optimized bulk description of files by replacing `system/describeDataObjects` with `system/findDataObjects` API call and scoping file search to projects
+* Made array type-checking less strict to coerce between empty and non-empty array types
+* Made it possible to run apps built using dxCompiler (after applets to apps conversion using dx-toolkit SDK, min. v0.296)
```

Comment:
what's that about?

Reply:
It's about the ability to run apps, not only applets, compiled from WDL.